cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+
+# OpenID Connect claim handlers
+OAUTH_OIDC_ID_TOKEN_HANDLERS = (
+    'oauth2_provider.oidc.handlers.BasicIDTokenHandler',
+    'oauth2_provider.oidc.handlers.ProfileHandler',
+    'oauth2_provider.oidc.handlers.EmailHandler',
+    # Includes the "administrator" claim in the token. This currently resides in lms/djangoapps!
+    'oauth2_handler.IDTokenHandler'
```

Comment:
I'm having trouble getting the "administrator" claim &mdash; provided by the "permissions" scope &mdash; into the ID tokens generated for use in the context of Studio. Prior to rebasing this PR onto Django 1.8, including this `IDTokenHandler` was enough to get the "administrator" claim. Without the "administrator" claim in the token, the Programs API won't return unpublished programs. I'd appreciate any help/suggestions.

This wasn't a problem prior to rebasing this onto Django 1.8.